meson.build

```diff
@@ -115,14 +116,20 @@ else
   dxvk_wsi = get_option('dxvk_native_wsi')
   
   if dxvk_wsi == 'sdl2'
-    lib_sdl2 = cpp.find_library('SDL2')
+    sdl2_dep = dependency('sdl2')
     compiler_args += ['-DDXVK_WSI_SDL2']
   elif dxvk_wsi == 'glfw'
-    lib_glfw = cpp.find_library('glfw')
+    glfw_dep = dependency('glfw3')
```

Comment:
This isn't working as I'd hoped yet: it's not generating a `Requires.private: sdl2` in the `.pc` files. I think that's probably because the shared libraries pick up their dependency on SDL/GLFW via a chain of static libraries, rather than explicitly depending on it. More adjustment needed there.